src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                <tr>
+                    <td>2</td>
+                    <td>
+                        Request info:
+                        <br></br>
+                        0 = GO serial number
+                        <br></br>
+                        1 = GO firmware version
+                        <br></br>
+                        2 = IOX protocol version
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>GO Status Information (0x26)</h3>
+        <p>Sent from the GO device to the IOX to pass information the IOX may need. This is a broadcast message. It is sent once any corresponding information type changes.</p>
+        <h4>Payload</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>Information Type</td>
+                </tr>
+                <tr>
+                    <td>2-7</td>
+                    <td>Optional bytes that depend on the type</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Information Type 0 - Ignition</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0000</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        0 = Ignition Off
+                        <br></br>
+                        1 = Ignition On
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Information Type 1 - Modem Availability</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0000</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        0 = Modem is not ready
+                        <br></br>
+                        1 = Modem is available
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>GO Multi-Frame Data (0x27)</h3>
+        <p>Sent from the GO device to the IOX when the GO device wants to transfer data that does not fit into a single CAN frame. The first frame contains the Type and Length. All frames start with a Frame Counter that is an incrementing sequence number. The first frame always starts with 0x00.</p>
+        <h4>Payload First Frame</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on